codes/quantum/qudits/qutrit_golay.yml

```diff
@@ -0,0 +1,40 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: qutrit_golay
+physical: qudits
+logical: qudits
+
+name: 'Qutrit Golay code'
+introduced: '\cite{arXiv:2003.02717}'
+
+description: |
+
+  The qutrit Golay code is an \([[11,1,5]]_3\) CSS quantum error-correcting code. The \(X\) and \(Z\) stabilizer generators can both be given by the following generator matrix
+  \begin{align}
+    \left(\begin{matrix}
+      2&1&1&2&2&0&1&0&0&0&0\\
+      2&1&2&1&0&2&0&1&0&0&0\\
+      2&2&1&0&1&2&0&0&1&0&0\\
+      2&2&0&1&2&1&0&0&0&1&0\\
+      2&0&2&2&1&1&0&0&0&0&1
+    \end{matrix}\right).
+  \end{align}
+  In the above matrix, each row encodes an \(X\) and a \(Z\) stabilizer generator where the qutrit Pauli \(X\) and \(Z\) operators are defined by \(X|k\rangle=|k+1\rangle\) and \(Z|k\rangle=\omega^k|k\rangle\) for every \(k\in \mathbb{Z}_3\) and \(\omega=e^{2\pi/3 i}\)\cite{arXiv:2003.02717}.
+
+features:
+

```

Comment:
I'd like this field to be present so that this code can be included in the list https://errorcorrectionzoo.org/list/quantum_magic. To do so, please add magic_scaling_exponent field for the related magic-state scheme. If a specific exponent doesn't make sense, please still add a description of the magic-state protocol.